app/utils/query_helpers.py

```diff
@@ -116,31 +117,39 @@ def query_count(query: Query) -> int:
 
 
 @functools.cache
-def get_stat_domains() -> dict[str, type[DeclarativeBase]]:
-    return {
-        "overall": DeadlockedOverallStats,
-        "deathmatch": DeadlockedDeathmatchStats,
-        "conquest": DeadlockedConquestStats,
-        "ctf": DeadlockedCTFStats,
-        "koth": DeadlockedKOTHStats,
-        "juggernaut": DeadlockedJuggernautStats,
-        "weapon": DeadlockedWeaponStats,
-        "vehicle": DeadlockedVehicleStats,
-
-        "horizon": DeadlockedHorizonStats,
-        "snd": DeadlockedSNDStats,
-        "payload": DeadlockedPayloadStats,
-        "spleef": DeadlockedSpleefStats,
-        "infected": DeadlockedInfectedStats,
-        "gungame": DeadlockedGungameStats,
-        "infinite_climber": DeadlockedInfiniteClimberStats,
-        "survival": DeadlockedSurvivalStats,
-        "survival_orxon": DeadlockedSurvivalOrxonStats,
-        "survival_mountain_pass": DeadlockedSurvivalMountainPassStats,
-        "survival_veldin": DeadlockedSurvivalVeldinStats,
-        "training": DeadlockedTrainingStats,
-    }
-
+def get_stat_domains(game: str) -> dict[str, type[DeclarativeBase]]:
```

Comment:
I think it would be good to make the function names unique. I'll do the same on the DL side.